htdocs/js/ui/init.js

```diff
@@ -67,7 +67,9 @@ RCloud.UI.init = function() {
     RCloud.UI.pull_and_replace.init();
 
     // incremental search
-    RCloud.UI.incremental_search.init();
+    if(!shell.is_view_mode()) {
+        RCloud.UI.incremental_search.init();
+    }
 
```

Comment:
I guess this is for #2478 - I think I'll go ahead and merge that along with this.